docs/_guide/attrs.md

```diff
@@ -0,0 +1,204 @@
+---
+chapter: 7
+subtitle: Using attributes as configuration
+---
+
+Components may sometimes manage state, or configuration. We encourage the use of DOM as state, rather than maintaining a separate state. One way to maintain state in the DOM is via [Attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes).
+
+As Catalyst elements are really just Web Components, they have the `hasAttribute`, `getAttribute`, `setAttribute`, `toggleAttribute`, and `removeAttribute` set of methods available, as well as [`dataset`](https://developer.mozilla.org/pl/docs/Web/API/HTMLElement/dataset), but these can be a little tedious to use; requiring null checking code with each call.
```

Comment:
Please, change the language of the link to `developer.mozilla.org` from `pl` to `en-US`

Reply:
Thanks! Nice catch!

```suggestion
As Catalyst elements are really just Web Components, they have the `hasAttribute`, `getAttribute`, `setAttribute`, `toggleAttribute`, and `removeAttribute` set of methods available, as well as [`dataset`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/dataset), but these can be a little tedious to use; requiring null checking code with each call.
```